aerospike_helpers/expressions/map.py

```diff
@@ -223,13 +223,14 @@ class MapRemoveByKeyList(_BaseExpr):
 
     _op = aerospike.OP_MAP_REMOVE_BY_KEY_LIST
 
-    def __init__(self, ctx: "TypeCTX", keys: List[TypeKey], bin: "TypeBinName"):
+    def __init__(self, ctx: "TypeCTX", keys: List[TypeKey], bin: "TypeBinName", inverted: bool = False):
         """Args:
             ctx (TypeCTX): An optional list of nested CDT :mod:`cdt_ctx <aerospike_helpers.cdt_ctx>` context operation
                 objects.
             key (List[TypeKey]): List of key values or a list expression of keys to elements to remove.
             bin (TypeBinName): bin expression, such as :class:`~aerospike_helpers.expressions.base.MapBin` or
                 :class:`~aerospike_helpers.expressions.base.ListBin`.
+            inverted (bool): Return the opposite expected results of map command.
```

Comment:
nit: same as in list, can you reword this?

Reply:
Fixed